driver/registry_default_hooks.go

```diff
@@ -67,7 +70,12 @@ func (m *RegistryDefault) getHooks(credentialsType string, configs []config.Self
 		case hook.KeySessionDestroyer:
 			i = append(i, m.HookSessionDestroyer())
 		case hook.KeyWebHook:
-			i = append(i, hook.NewWebHook(m, h.Config))
+			cfg := request.Config{}
+			if err := json.Unmarshal(h.Config, &cfg); err != nil {
+				m.l.WithError(err).WithField("raw_config", string(h.Config)).Error("failed to unmarshal hook configuration, ignoring hook")
```

Comment:
This is opaque to the user and at the very least needs an event? Before webhook mis config would cause an error in the request chain, now it’s silently ignored which makes webhooks even nastier to configure in network IMO. Maybe move this back to the request chain?

Reply:
OK, I re-added this to the request chain now.